_search-plugins/search-pipelines/normalization-processor.md

```diff
@@ -32,6 +32,9 @@ The following table lists all available request fields.
 Field | Data type | Description
 :--- | :--- | :---
 `normalization.technique` | String | The technique for normalizing scores. Valid values are [`min_max`](https://en.wikipedia.org/wiki/Feature_scaling#Rescaling_(min-max_normalization)) and [`l2`](https://en.wikipedia.org/wiki/Cosine_similarity#L2-normalized_Euclidean_distance). Optional. Default is `min_max`.
+`normalization.parameters.lower_bounds` | Array of objects | Each object in array defines the details of lower bound for one query. The number of values in the `lower_bounds` array must equal the number of queries. Optional. Only applicable when normalization technique is [`min_max`](https://en.wikipedia.org/wiki/Feature_scaling#Rescaling_(min-max_normalization)). If not provided, lower bound will not be applied for any of the sub-queries and system will use actual minimum score. Introduced in 3.0.
+`normalization.parameters.lower_bounds.mode` | String | The mode of how lower bound is used for one query. Valid values are `apply` - use min_score for normaliation but do not replace original scores, `clip` - replace scores below lower bound with the min_score value, `ignore` - do not apply lower bound for this query. Optional. Default is `apply`.
+`normalization.parameters.lower_bounds.min_score` | Number | Floating-point number that is used as a lower bound treshold. Valid values are in the [-10000.0, 10000.0] range. If `mode` is defined as `ignore` then `min_score` value will not have any effect. Optional. Default is 0.0.
```

Comment:
Isn't default is `0.001`?

Reply:
nope, default lower_bound min_score is 0.0. This is not the actual score, but the lower bound for the interval that we use to scale the feature